dev-lisp/sbcl/sbcl-2.5.4.ebuild

```diff
@@ -178,6 +181,8 @@ src_compile() {
 	if use system-bootstrap ; then
 		if has_version "dev-lisp/sbcl" ; then
 			bootstrap_lisp="sbcl --no-sysinit --no-userinit --disable-debugger"
+		elif use elibc_musl ; then
```

Comment:
If we are to do away with `elibc_musl` here, and I hope we can, then this should be tweaked to be a case statement with some has_versions, I think.

Reply:
Agree, that seems like a better solution.